src/index.js

```diff
@@ -125,6 +126,41 @@ class Mongo {
     }
   }
 
+  async generateVerificationEmailToken(userId: string, email: string): Promise<void> {
+    const ret = await this.collection.update({ _id: userId }, {
+      $push: {
+        'services.email.verificationTokens': {
+          token: crypto.randomBytes(43).toString('hex'),
```

Comment:
Yes, the token should be generated in `@accounts/server` and then passed down to the database method call.